lib/modules/manager/kotlin-script/index.ts

```diff
@@ -0,0 +1,14 @@
+export { extractPackageFile } from './extract';
+
+import { ProgrammingLanguage } from '../../../constants';
+import { MavenDatasource } from '../../datasource/maven';
+import * as gradleVersioning from '../../versioning/gradle';
+
+export const language = ProgrammingLanguage.Kotlin;
+
+export const defaultConfig = {
+  fileMatch: ['^.*\\.kts$'],
```

Comment:
we should by default only support `.main.kts` files. a user can always override with config.